### DISCUSSION TITLE
Combobox dropdown has no items, despite items being actually presented

### DISCUSSION THREAD

**Discussion by ZecosMAX** [AvaloniaUI/Avalonia#10103] [Q&A] [answered]
I have following XAML code in my `MainWindow.xaml`:
```XAML
<ComboBox SelectedIndex="0">
    <ComboBoxItem>Item 1</ComboBoxItem>
    <ComboBoxItem>Item 2</ComboBoxItem>
    <ComboBoxItem>Item 3</ComboBoxItem>
    <ComboBoxItem>Item 4</ComboBoxItem>
    <ComboBoxItem>Item 5</ComboBoxItem>
</ComboBox>
```
And a dark fluent theme in `App.xaml`: 
```XAML
<Application.Styles>
    <FluentTheme Mode="Dark"/>
</Application.Styles>
```
And when application is running, combobox's dropdown has no items inside of it:
![image](https://user-images.githubusercontent.com/20259054/215078669-c1559a09-9013-4ee6-b3f4-de34283e7d83.png)

Despite that, when combobox is in focus, i can still use keyboard arrows to navigate through items, meaning they are presented in control.
![image](https://user-images.githubusercontent.com/20259054/215078880-d18f4734-c2d8-4421-9cbb-549019f3bb19.png)

I expected items being presented inside dropdown.

- OS: Windows 10.0 build 19045
- Version: 0.10.18

**Comment by ZecosMAX** [comment]
I have more context, when ComboBox's property `IsDropDownOpen` is set to `true`, for some reason item presenter has property `IsVisible` set to `false` by `Style`. Setting it manually to `true` fixes things

![image](https://user-images.githubusercontent.com/20259054/215082002-d6ec18d3-4e3c-4eeb-b8a0-2af2aa547801.png)

so i have the workaround, setting style of combobox like this fixes this.
```XAML
<ComboBox.Styles>
    <Style Selector="ItemsPresenter#PART_ItemsPresenter">
        <Setter Property="IsVisible" Value="True"/>
    </Style>
</ComboBox.Styles>
```

A crude fix, it doesn't seem to be intended behavior to me

**Comment by timunie** [comment]
Can you please prepare a minimal sample app showing your issue? I really think there must be a custom style or whatever that does this. It's not a known issue. And I'm sure if that's by default, someone would have complained already.

**Comment by ZecosMAX** [comment, marked as answer]
@timunie false alarm, yes, this was a fault in our project. This `ComboBox` was inside of a `TabControl` which has following styles:
```
<TabControl.Styles>
    <Style Selector="ItemsPresenter#PART_ItemsPresenter">
        <Setter Property="IsVisible" Value="False"/>
    </Style>
</TabControl.Styles>
```

So the default header would not be visible. 
And the style selector affected nested control.
We (me in particular) failed to notice that, sorry for that :(
